keycloak/keycloak-admin-ui
# Event listeners

## Motivation
- Adding event listeners sub-tab as per https://marvelapp.com/prototype/6ac0aja/screen/83306311
- Adding cypress tests

## Verification Steps
1.  Go to /realm-settings/events tab and verify that the form is as per design. 
2. Select an option from the dropdown and select "Revert". Revert should revert to the original/ default selection if any.
3. Select an option from the dropdown and select "Save". Success alert should display. Verify that saved events are now visible.

## Checklist:
